Cubical/Categories/Constructions/Free/Functor.agda

```diff
@@ -0,0 +1,346 @@
+                  substOf-sems-agreeϕ =
+                    sym (cong-∙ A (λ i x → 𝓕 ⟅ arb𝓒-agree (~ i) $g x ⟆)
+                                λ i x → arb𝓕 i ⟅ x ⟆)
+                    ∙ cong (cong A)
+                           (Square→compPath λ i j x → arb𝓕-agree (~ i) j x)
+                    ∙ cong-∙ A (λ i x → ıϕp i x)
+                             (λ i x → arb𝓓-agree (~ i) $g ϕ x) where
+                      the-type = (G .Node → 𝓓 .ob)
+                      A = (λ (f : the-type) → 𝓓 [ f v , f w ])
+          aom : ∀ {v w : H .Node} (f : FH [ v , w ]) → aom-type f
+          aom (↑ x) = λ i → arb𝓓-agree i <$g> x
+          aom idₑ = aom-id
+          aom (f ⋆ₑ g) = aom-seq (aom f) (aom g)
+          aom F⟪ x ⟫ = aom-F x
+          aom (⋆ₑIdL f i) =
+            isSet→SquareP
+              (λ i j → 𝓓 .isSetHom)
+              (aom-seq aom-id (aom f))
+              (aom f)
+              (λ i → arb𝓓 ⟪ ⋆ₑIdL f i ⟫)
+              (λ i → (semH ⟪ ⋆ₑIdL f i ⟫))
+              i
+          aom (⋆ₑIdR f i) =
+            isSet→SquareP (λ i j → 𝓓 .isSetHom)
+              (aom-seq (aom f) aom-id)
+              (aom f )
+              (λ i → arb𝓓 ⟪ ⋆ₑIdR f i ⟫)
+              (λ i → semH ⟪ ⋆ₑIdR f i ⟫)
+              i
+          aom (⋆ₑAssoc f f₁ f₂ i) =
+              isSet→SquareP
+                (λ i j → 𝓓 .isSetHom)
+                (aom-seq (aom-seq (aom f) (aom f₁)) (aom f₂))
+                (aom-seq (aom f) (aom-seq (aom f₁) (aom f₂)))
+                (λ i → arb𝓓 ⟪ ⋆ₑAssoc f f₁ f₂ i ⟫)
+                (λ i → semH ⟪ ⋆ₑAssoc f f₁ f₂ i ⟫)
+                i
+          aom (F-idₑ i) =
+            isSet→SquareP
+              (λ i j → 𝓓 .isSetHom)
+              (aom-F idₑ)
+              aom-id
+              (λ i → arb𝓓 ⟪ F-idₑ i ⟫)
+              (λ i → semH ⟪ F-idₑ i ⟫)
+              i
+          aom (F-seqₑ f g i) =
+            isSet→SquareP
+              (λ i j → 𝓓 .isSetHom)
+              (aom-F (f ⋆ₑ g))
+              (aom-seq (aom-F f) (aom-F g))
+              (λ i → arb𝓓 ⟪ F-seqₑ f g i ⟫)
+              (λ i → semH ⟪ F-seqₑ f g i ⟫)
+              i
+          aom (isSetFExp f f₁ x y i j) k =
+            isSet→SquareP
+              (λ i j → (isOfHLevelPathP {A = λ k → 𝓓 [ aoo _ k , aoo _ k ]} 2
+                                        (𝓓 .isSetHom)
+                                        (arb𝓓 ⟪ isSetFExp f f₁ x y i j ⟫)
+                                        ((semH ⟪ isSetFExp f f₁ x y i j ⟫))))
+              (λ j k → aom (x j) k)
+              (λ j k → aom (y j) k)
+              (λ i k → aom f k)
+              (λ i k → aom f₁ k)
+              i j k
+
+        -- TODO
+        -- sem-uniq-ϕ : Square arb𝓕
+        --                     (idToPath semϕ)
+        --                     (λ i → 𝓕 ∘F sem-uniq-G i)
+        --                     (λ i → sem-uniq-H i ∘F Fϕ)
+        -- sem-uniq-ϕ = {!!}
+
+        -- TODO: uniqueness of the uniqueness paths above
```

Comment:
It would be good to give more context, if this TODO is left. I think, in general it is better to give a summary comment at the start of the file, what has been done and what is still missing.